pkg/repositories/config/postgres.go

```diff
@@ -2,21 +2,33 @@ package config
 
 import (
 	"fmt"
+	"time"
 
 	"github.com/flyteorg/flytestdlib/promutils"
 
 	"github.com/jinzhu/gorm"
 	_ "github.com/jinzhu/gorm/dialects/postgres" // Required to import database driver.
 )
 
-const Postgres = "postgres"
+const (
+	Postgres                     = "postgres"
+	DefaultMaxIdleConnection     = 25
```

Comment:
instead of setting these defaults here, can we set them in the runtime config when calling MustRegisterSection like so: https://github.com/flyteorg/flyteadmin/blob/master/pkg/runtime/application_config_provider.go#L25,L30

Reply:
ok